### DISCUSSION TITLE
How to Switch from the develop to master repository

### DISCUSSION THREAD

**Discussion by archedraft** [TandoorRecipes/recipes#2961] [Q&A] [answered]
I am attempting to switch from  the vabene1111/recipes:develop repository to the master. I am able to switch to master and open the app once it starts. I can see the recipes but if I click on any recipe, I get the following error and nothing loads.
`There was an error loading a resource! "\n<!doctype html>\n<html lang=\"en\">\n<head>\n <title>Server Error (500)</title>\n</head>\n<body>\n <h1>Server Error (500)</h1><p></p>\n</body>\n</html>\n"`

What's the best way to accomplish this?

![image](https://github.com/TandoorRecipes/recipes/assets/7254021/bbd0f2ba-d86a-48ba-b7be-0df9a1d483d8)

**Comment by smilerz** [comment, marked as answer]
There is frequently no way to downgrade.  The safest path is to wait for the next release and then switch.  

The other method is to restore the db from backup before switching to develop. 

Or, for the expert, manually back out whatever db schema changes have taken place.

**Reply by archedraft** [reply]
Understood, thanks for the reply!
